sdl2_image/pkg-config-fix.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 438a0aa..6bce3d6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -694,37 +694,34 @@ if(SDL2IMAGE_INSTALL)
+         COMPONENT devel
+     )
+ 
+-    if(SDL2IMAGE_BUILD_SHARED_LIBS)
```

Comment:
it's not important things, how about just adding the `#` to if block instead reduce all indent?
i couldn't remember, but cmake doesn't require strict indentation.

Reply:
It's formatted like this because I cherry-picked this commit: https://github.com/libsdl-org/SDL_image/commit/ae87511a72af8ab3e8af9ba69f5d9af5d5d80a78

CMake does not care about indentation.

Should I change this?

Reply:
oh...... can you change it too?
moving all indentations could lead failing patch with future versions.
but if you don't think about it, it's okay. just share your opinion. :)

Reply:
I doubt there will be another release of SDL2_image, since the next release of SDL will be SDL3. I can change it, though. I got some time either tonight or tomorrow.

Reply:
@d3m3vilurr I've undone the indentation change and made the patch smaller. Hope that works better for you guys.